scripts/wasmd/send_first.js

```diff
@@ -20,8 +25,9 @@ async function main() {
   const client = await SigningStargateClient.connectWithSigner(rpcUrl, wallet, { prefix: prefix });
   const recipient = Bech32.encode(prefix, Random.getBytes(20));
   const amount = coins(226644, "ucosm");
+  const fee = calculateFee(80_000, GasPrice.fromString("0.025ucosm"));
```

Comment:
```suggestion
  const fee = calculateFee(80_000, "0.025ucosm");
```

What about accepting `GasPrice | string` to make this simple case more compact?